### ISSUE TITLE
Feature request: export all shaders as ZIP

### ISSUE THREAD

**Issue by cxw42** [patuwwy/ShaderToy-Chrome-Plugin#49]
Add to the profile page an "Export all shaders as ZIP" option.  This complements the existing ability to export all shaders as JSON (#24).

Blocked on #47 per discussion with @patuwwy earlier today.

**Comment by cxw42** [comment]
@patuwwy Do you still want the profile-page option?  It's not in #47 because you asked me to keep the PR focused.  I was going to work on this next, but if you are no longer interested, I won't!

**Comment by patuwwy** [comment]
Sure. Please do this If You want to.
It shouldn't be hard to implement, You can extract some methods in existing code - there is already functionality to export all shaders in one JSON (by @movAX13h )

**Comment by cxw42** [comment]
Yes, I would like to; I think it would be useful to have.  Would you please reopen this issue so it stays on my dashboard?  Thanks!
